app/views/repos/_script.html.haml

```diff
@@ -5,5 +5,6 @@
     = link_to benchmark_type.github_url, class: 'btn btn-primary', target: '_link' do
       = t('repos.view_github')
 
+  - require 'net/http'
```

Comment:
Why would you need this? It's run in the context of Rails where net/http is already required.